eng/common/scripts/artifact-metadata-parsing.ps1

```diff
@@ -406,7 +406,7 @@ function IsPythonPackageVersionPublished($pkgId, $pkgVersion) {
 # Retrieves the list of all tags that exist on the target repository
 function GetExistingTags($apiUrl) {
   try {
-    return (Invoke-WebRequest -Method "GET" -Uri "$apiUrl/git/refs/tags"  -MaximumRetryCount 3 -RetryIntervalSec 10) | % { $_.ref.Replace("refs/tags/", "") }
+    return (Invoke-RestMethod -Method "GET" -Uri "$apiUrl/git/refs/tags"  -MaximumRetryCount 3 -RetryIntervalSec 10) | % { $_.ref.Replace("refs/tags/", "") }
```

Comment:
Looks like we were calling `Invoke-WebRequest-WithHandling` before the most recent change. The change you made works locally for me.

Reply:
Yep. I saw that too. I'm not certain how this boyo slipped through testing, but it's a super minor update and `Invoke-WebRequest-WithHandling` used `Invoke-RestMethod` internally anyway 👍 

Glad to see you saw the behavior well. Definitely gives me confidence :)